src/doc/src/reference/unstable.md

```diff
@@ -1215,6 +1215,30 @@ cargo check -Z unstable-options -Z check-cfg=features,names,values
 
 * RFC: [#2906](https://github.com/rust-lang/rfcs/blob/master/text/2906-cargo-workspace-deduplicate.md)
 * Tracking Issue: [#8415](https://github.com/rust-lang/cargo/issues/8415)
+* [Current Summary](https://github.com/rust-lang/cargo/issues/8415#issuecomment-1112618913)
+* [Example Port](https://github.com/clap-rs/clap/pull/3719)
+
+### Testing notes
+
+We are looking for people to test `workspace-inheritance` and provide feedback about it.
+
+We would like to know
+* If there were any pain points in porting your project
+* Any errors or bugs that you found in testing
+* Performance concerns
+* Gaps in documentation
+
+Anything that you find please let us know by filing an issue or commenting on
+the [tracking issue](https://github.com/rust-lang/cargo/issues/8415).
+
+To get started make sure you are running a nightly version of cargo. After that 
```

Comment:
Something thats missing is setting the expectations for testing, that what we are looking for ranges from people doing test-ports to people adopting it who are running nightly.  We need to make it clear that we still think its useful for someone to test this without switching to nightly.  We need to set this expectation early and quickly or people will assume they need to be developing on nightly to help out.

Reply:
Should I add something similar to what I put below, above this?

"You do not have to be running nightly in you project currently or switch to nightly in order help out. You can do a test port [like was done here](https://github.com/clap-rs/clap/pull/3719) and that will help us a lot!"

Reply:
Depending on how it gets inserted, that could work